{"name": "新增客户失败-手机号格式不正确", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\nrun.py:121: AssumptionFailure\n>>\tpytest.assume(actual_res == expect_res, f'期望是{expect_res},实际是{actual_res}')\nAssertionError: 期望是1,实际是0\nassert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               run.py:121: AssumptionFailure\nE               >>\tpytest.assume(actual_res == expect_res, f'期望是{expect_res},实际是{actual_res}')\nE               AssertionError: 期望是1,实际是0\nE               assert False\n\nC:\\Python37\\lib\\site-packages\\six.py:702: FailedAssumption"}, "attachments": [{"name": "log", "source": "6fa66b70-3e21-4508-9b85-451200800cbb-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "suite_name", "value": "'新增客户接口测试集合'"}, {"name": "case_name", "value": "'新增客户失败-手机号格式不正确'"}, {"name": "case_info_list", "value": "[['新增客户', '${host}/CrmCustomer/addOrUpdate', 'post', '{\\n\"Admin-Token\":\"${token}\"\\n}', '{\\n\"json\":{\\n\"$.entity.mobile\":\"sddsdffd\",\\n}\\n}', '', 200, '[\\n{\\n\"actual\":\"$.code\",\\n\"expect\":1\\n}\\n]']]"}], "start": 1661766072681, "stop": 1661766073294, "uuid": "1475d9e2-5755-4577-b33f-4a570a915a26", "historyId": "765634552f43e06b8aef2099250e5907", "testCaseId": "581fcf0b30a85e3d611d5f9973144474", "fullName": "run#test_run", "labels": [{"name": "feature", "value": "新增客户接口测试集合"}, {"name": "suite", "value": "run"}, {"name": "host", "value": "N-5CG8250K26"}, {"name": "thread", "value": "19024-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "run"}]}